src/ig/IGExporter.ts

```diff
@@ -193,15 +195,27 @@ export class IGExporter {
     const inputPath = path.join(__dirname, 'files');
     this.copyAsIs(inputPath, igPath);
 
+    // If in an IG Publisher context, do not include any of the publisher scripts
+    if (this.isIgPubContext) {
+      removeSync(path.join(igPath, '_genonce.sh'));
+      removeSync(path.join(igPath, '_genonce.bat'));
+      removeSync(path.join(igPath, '_gencontinuous.sh'));
+      removeSync(path.join(igPath, '_gencontinuous.bat'));
+      removeSync(path.join(igPath, '_updatePublisher.sh'));
+      removeSync(path.join(igPath, '_updatePublisher.bat'));
+    }
```

Comment:
I'm not sure we want to _remove_ these.  What if the user _wants_ these scripts at the root of their repo and copies them there themselves?  We don't want to delete the ones the user added themselves.  I think we need to make the copy smarter (by applying a filter or something)...

Reply:
Ohh that's a good point. I'll look into this.